### ISSUE TITLE
Setup Dev Env for Golang

### ISSUE THREAD

**Issue by uxjp** [uxjp/ready_envs#7]

**Comment by uxjp** [comment]
```bash 
@root app
|> wget https://go.dev/dl/go1.22.5.linux-arm64.tar.gz
--2024-07-31 13:40:48--  https://go.dev/dl/go1.22.5.linux-arm64.tar.gz
Resolving go.dev (go.dev)... 216.239.38.21, 216.239.32.21, 216.239.34.21, ...
Connecting to go.dev (go.dev)|216.239.38.21|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://dl.google.com/go/go1.22.5.linux-arm64.tar.gz [following]
--2024-07-31 13:40:48--  https://dl.google.com/go/go1.22.5.linux-arm64.tar.gz
Resolving dl.google.com (dl.google.com)... 142.251.135.142, 2800:3f0:4004:809::200e
Connecting to dl.google.com (dl.google.com)|142.251.135.142|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 65911806 (63M) [application/x-gzip]
Saving to: 'go1.22.5.linux-arm64.tar.gz'

go1.22.5.linux-arm 100%[================>]  62.86M  5.93MB/s    in 12s     

2024-07-31 13:41:01 (5.15 MB/s) - 'go1.22.5.linux-arm64.tar.gz' saved [65911806/65911806]

@root app
|> ls
go1.22.5.linux-arm64.tar.gz
@root app
|> tar -C /usr/local -xzf go1.22.5.linux-arm64.tar.gz 
@root app
|> vi ~/.profile 
@root app
|> wget https://go.dev/dl/go1.20.6.linux-arm64.tar.gz^C
@root app
|> echo "export PATH=\$PATH:/usr/local/go/bin" >> ~/.profile
@root app
|> source ~/.profile
@root app
|> go version
go version go1.22.5 linux/arm64
@root app
|> vi
@root app
|> vi hello.go
@root app
|> go run hello.go 
hello, world
@root app

```

**Comment by uxjp** [comment]
Test Passing
<img width="846" alt="Screenshot 2024-07-31 at 12 25 16" src="https://github.com/user-attachments/assets/229f01ff-33ee-49ee-ba67-6f2f28e66c19">

**Comment by uxjp** [comment]
closes it #8
